packages/nextjs/src/app-router/client/ClerkProvider.tsx

```diff
@@ -24,6 +26,12 @@ declare global {
 }
 
 export const ClientClerkProvider = (props: NextClerkProviderProps) => {
+  if (nextPackage.version.startsWith('13.') || nextPackage.version.startsWith('14.0')) {
```

Comment:
```suggestion
  if (nextPackage.version.startsWith('13.') || nextPackage.version.startsWith('14.0') || nextPackage.version.startsWith('14.1')) {
```

I assume since we're suggesting that the user upgrade to >= 14.2 that we're also dropping support for 14.1?

Reply:
hmmm i think i might have a typo there. I found that 14.1 works as expected